app/models/relation.rb

```diff
@@ -101,6 +101,9 @@ class Relation < ApplicationRecord
 
   ORDERED_TYPES = [*TYPES.keys, TYPE_PARENT, TYPE_CHILD].freeze
 
+  MIN_LAG = -2_000
+  MAX_LAG = 2_000
```

Comment:
Very minor, but since these are additive inverses:

```suggestion
  MAX_LAG = 2_000
  MIN_LAG = -MAX_LAG
```

Reply:
Good point.